### PR TITLE
I1932 process schedules for lab

### PR DESCRIPTION
**Pull request by gabriel-happle**
This was supposed to be a small issue.
Many other things were found and fixed.
- Process schedules added to LAB and HOSPITAL in SG (mirroring CH)
- reading of all currently present process schedules in occupancy model (used to be only INDUSTRIAL)
- processing internal loads and other archetype information for relatively new uses of  'LAB', 'MUSEUM', 'LIBRARY', 'UNIVERSITY' (they were just ignored previously)
- creating an exception for unknown occupancies in `occupancy.dbf`

assigning @martin-mosteiro for review and @daren-thomas because of the exception.

### CODE REVIEW

**Comment by gabriel-happle** [comment]
To test the intended behavior, you can use this modified `occupancy.dbf` for the `reference-case-cooling`.
Building `B005` is modified to 10% LAB use and therefore should in the end have an `Epro_kWh > 0`.
https://drive.google.com/open?id=1vB1-6_LXMWN29QnGSCsPrIOQ4qiW2H_B

To test the Exception, you can use a occupancy.dbf with a typo in one of the uses.

File: cea/datamanagement/databases_verification.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 COLUMNS_DISTRICT_GEOMETRY = ['Name', 'height_ag', 'floors_ag']
 COLUMNS_ZONE_AGE = ['built', 'roof', 'windows', 'partitions', 'basement', 'HVAC', 'envelope']
 COLUMNS_ZONE_OCCUPANCY = ['MULTI_RES', 'OFFICE', 'RETAIL', 'SCHOOL', 'HOTEL', 'GYM', 'HOSPITAL', 'INDUSTRIAL',
-                          'RESTAURANT','SINGLE_RES', 'SERVERROOM', 'SWIMMING', 'FOODSTORE', 'LIBRARY', 'COOLROOM', 'PARKING']
+                          'RESTAURANT','SINGLE_RES', 'SERVERROOM', 'SWIMMING', 'FOODSTORE', 'LIBRARY', 'COOLROOM',
+                          'PARKING', 'LAB', 'MUSEUM', 'LIBRARY', 'UNIVERSITY']
```
Comment (martin-mosteiro) [inline cea/datamanagement/databases_verification.py]: Didn't know there was a 'UNIVERSITY' archetype in SG (in CH we have just used 'SCHOOL' for lecture halls and specific archetypes for any other uses). Are these schedules taken from ASHRAE?

Just curious, no action needed.

**Comment by martin-mosteiro** [comment]
Errata: the above commit *does* change the demand for domestic hot water because previously FOODSTORE had no hot water demand on Saturdays even though there were occupants on those days. That was a mistake, which has now been fixed.

File: cea/datamanagement/databases_verification.py (hunk 1 above)
Comment (gabriel-happle) [reply cea/datamanagement/databases_verification.py]: I think most of the schedules for SG are copied from CH.

**Comment by gabriel-happle** [comment]
@martin-mosteiro what did you change exactly? 
What does it mean to "make the DHW schedules consistent across all functions?
If you think the PR works as intended, i.e., all functions are now considered, all schedules are read and processed,... I suggest we merge it.
I can create other issues about the values in the schedules.
I already created #2030.
I saw you created #2051.

**Comment by martin-mosteiro** [comment]
>  What does it mean to "make the DHW schedules consistent across all functions?
The dhw schedules in CEA are processed as follows:
```
Schedule_dhw,weekday (t) = P_dhw,weekday(t) / (P_dhw,weekday(0) + ... + P_dhw,weekday(24))
```
where P_dhw is the dhw schedule from the occupancy database and Schedule is the actual schedule read by CEA.
For all occupancy types except residential, however, the schedule in the occupancy file is either `P_dhw,weekday(t) = P_occ,weekday(t)` **OR** `P_dhw,weekday = P_occ,weekday(t) / (P_occ,weekday(0) + ... + P_occ,weekday(24))`
So the results of either schedule are the same, but in one case the schedule looks like the occupancy schedule and in the other it looks like some obscure other number. I think it’s more transparent to keep P_dhw = P_occ for all uses except residential.

>  If you think the PR works as intended, i.e., all functions are now considered, all schedules are read and processed,... I suggest we merge it.
Fair enough, I'll turn the process schedule discussion into its own issue.

**Comment by gabriel-happle** [comment]
So I should probably create another issue about the DHW schedules in SG, @martin-mosteiro ?

**Comment by martin-mosteiro** [comment]
I don't think so. It's mainly a cosmetic/clarification issue. I can take care of it myself.

**Comment by martin-mosteiro** [comment]
Reverted the changes to the CH process schedules. All tests pass locally on my computer, but the following comes up on the Jenkins:
```
GitHub pull request #2034 of commit dc221e48a60fc2ab7d52e521aafab3c5e4328faf, no merge conflicts.
Setting status of dc221e48a60fc2ab7d52e521aafab3c5e4328faf to PENDING with url https://ceajenkins.ngrok.io/job/run%20cea%20test%20for%20pull%20requests/1641/ and message: 'Build started for merge commit.'
Building in workspace C:\Program Files (x86)\Jenkins\workspace\run cea test for pull requests
[WS-CLEANUP] Deleting project workspace...
[WS-CLEANUP] Deferred wipeout is used...
ERROR: [WS-CLEANUP] Cannot delete workspace: Unable to delete 'C:\Program Files (x86)\Jenkins\workspace\run cea test for pull requests'. Tried 3 times (of a maximum of 3) waiting 0.1 sec between attempts.
ERROR: Cannot delete workspace: Unable to delete 'C:\Program Files (x86)\Jenkins\workspace\run cea test for pull requests'. Tried 3 times (of a maximum of 3) waiting 0.1 sec between attempts.
Setting status of dc221e48a60fc2ab7d52e521aafab3c5e4328faf to FAILURE with url https://ceajenkins.ngrok.io/job/run%20cea%20test%20for%20pull%20requests/1641/ and message: 'Build finished. '
Finished: FAILURE
```
Doesn't seem to be a PR-related issue, so this PR would in principle be ready to merge (pending Jenkins approval).

**Comment by gabriel-happle** [comment]
@martin-mosteiro this happened before. It was resolved when @daren-thomas restarted the jenkins.

**Pull request merged**